classes/uuu_bootloader_tag.bbclass

```diff
@@ -4,8 +4,7 @@
 do_deploy:append() {
     if [ "${UUU_BOOTLOADER}" != "" ]; then
         cp ${DEPLOYDIR}/${UUU_BOOTLOADER}   ${DEPLOYDIR}/${UUU_BOOTLOADER_TAGGED}
-        ln -sf ${UUU_BOOTLOADER_TAGGED}     ${DEPLOYDIR}/${UUU_BOOTLOADER}
-        stat -L -cUUUBURNXXOEUZX7+A-XY5601QQWWZ%sEND ${DEPLOYDIR}/${UUU_BOOTLOADER} \
-                                         >> ${DEPLOYDIR}/${UUU_BOOTLOADER}
+        stat -L -cUUUBURNXXOEUZX7+A-XY5601QQWWZ%sEND ${DEPLOYDIR}/${UUU_BOOTLOADER_TAGGED} \
+                                         >> ${DEPLOYDIR}/${UUU_BOOTLOADER_TAGGED}
```

Comment:
I believe that `UUU_BOOTLOADER` determines what gets in the image, so this change will cause an image that used to be tagged to no longer be tagged.

Can you just make an explicit copy of the untagged file with suffix `-untagged`?

Reply:
Or maybe we could force the explicit file with suffix `-tagged` and `-untagged`?

Reply:
thanks for the comments, correct this could break the builds. I've added an `untagged` option, please let me know.